### PR TITLE
Variable cleanup

### LINKED ISSUE #99: Use uint8_t in getAttitude()
Same reasoning as #98, but in the `getAttitude()` function

### LINKED ISSUE #101: use clear naming scheme in settings.h
in `settings.h`, all use of `unsigned int` should be replaced with `uint32_t`, this is more informative from a readability standpoint

### LINKED ISSUE #103: tick should always be positive
in `void loop()` `tick` should be declared as a `uint32_t` instead of `int`. This will force the tick to always be positive.

### LINKED ISSUE #104: replace unsigned int
in `void loop()`, tickTime should use `uint32_t` instead of `unsigned int`. This will help with clarity.

### LINKED ISSUE #105: Update run function params to match input
since tick times will now be in `uint32_t` form from #103, all uses of tick should switch to `uint32_t` as well. This means all declarations and definitions of `tick` in `main.ino` will need to be updated.

### LINKED ISSUE #106: decrease garbage data size
In `void loop()` we have a variable named `x` that is simply used as a placeholder for future code. We might as well make x as small as possible to minimize our space usage. we can do this by using `bool` instead of `int`. This will decrease the space used from 4 bytes to 1 byte.

### LINKED ISSUE #107: use uint64_t for time
We're currently using `unsigned long` for storing time, but it isn't clear how large that number actually is. (it's either 32 bits or 64).  To be safe we should switch to using `uint64_t` (64 bits). In `utils.h`, replace all occurrences of `unsigned long` with `uint64_t`.

### LINKED ISSUE #108: replace unsigned short
in `utils.h` we're using `unsigned short` to track flight state. It's not clear how much space this takes, so we should replace the datatype with `uint8_t`

### LINKED ISSUE #109: delete #define SEALEVELPRESSURE_HPA (1013.25)
this line of code can be found in `main.ino`. It doesn't do anything since we don't care about sea level so we can just delete it.

### LINKED ISSUE #111: remove extra variable
near the top of `main.ino`, there's an unused variable declared called `apogeeReached`. This can cause quite a bit of confusion because there's another variable with the same name used later on that is unrelated. This unused variable should be deleted. Also, the `//variables` comment can be deleted since there won't be any global variables defined at that location.

### LINKED ISSUE #120: Remove unnecessary inclusion
In `apogee.h` we include both `settings.h` and `utils.h`. It's not necessary to include `settings.h` because `utils.h` is already including that file within itself.

Therefore we can delete the `#include settings.h` line

### PR DESCRIPTION
**Pull request by Sharkatron**
## Description of Problem

There are some extraneous variables declared without clear purposes. There are also variables that use up more space than required. Finally, the variables need to have more precise declarations so we know how much size it uses.

## Solution

Deleting extraneous variables and changing the variable declaration to lower sizes accordingly with their intended size. Also adjusting some variable to have more precise declarations. For example, changing `unsigned int` to `uint_32`

## Testing

Compiled the code without any warnings or errors.

## Logs

```
Processing teensy41 (platform: teensy; board: teensy41; framework: arduino)
--------------------------------------------------------------------------------
Verbose mode can be enabled via `-v, --verbose` option
CONFIGURATION: https://docs.platformio.org/page/boards/teensy/teensy41.html
PLATFORM: Teensy (4.17.0) > Teensy 4.1
HARDWARE: IMXRT1062 600MHz, 512KB RAM, 7.75MB Flash
DEBUG: Current (jlink) External (jlink)
PACKAGES:
 - framework-arduinoteensy @ 1.157.220801 (1.57)
 - tool-teensy @ 1.157.0 (1.57)
 - toolchain-gccarmnoneeabi @ 1.50401.190816 (5.4.1)
Converting main.ino
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
LDF Modes: Finder ~ chain, Compatibility ~ soft
Found 97 compatible libraries
Scanning dependencies...
Dependency Graph
|-- EWMA @ 1.0.2
|-- Adafruit BMP3XX Library @ 2.1.2
|   |-- Adafruit BusIO @ 1.13.2
|   |   |-- Wire @ 1.0
|   |   |-- SPI @ 1.0
|   |-- Wire @ 1.0
|   |-- SPI @ 1.0
|-- Adafruit GPS Library @ 1.7.0
|   |-- SPI @ 1.0
|   |-- Wire @ 1.0
|-- SD @ 2.0.0
|   |-- SdFat @ 2.1.2
|   |   |-- SPI @ 1.0
|-- Wire @ 1.0
Building in debug mode
Compiling .pio\build\teensy41\src\main.ino.cpp.o
Compiling .pio\build\teensy41\lib144\EWMA\Ewma.cpp.o
Compiling .pio\build\teensy41\lib631\Wire\Wire.cpp.o
Compiling .pio\build\teensy41\lib631\Wire\WireIMXRT.cpp.o
Compiling .pio\build\teensy41\lib631\Wire\WireKinetis.cpp.o
Compiling .pio\build\teensy41\lib631\Wire\utility\twi.c.o
Compiling .pio\build\teensy41\libf97\SPI\SPI.cpp.o
Compiling .pio\build\teensy41\libdd6\Adafruit BusIO\Adafruit_BusIO_Register.cpp.o
Compiling .pio\build\teensy41\libdd6\Adafruit BusIO\Adafruit_I2CDevice.cpp.o
Compiling .pio\build\teensy41\libdd6\Adafruit BusIO\Adafruit_SPIDevice.cpp.o
Compiling .pio\build\teensy41\lib59c\Adafruit BMP3XX Library\Adafruit_BMP3XX.cpp.o
Compiling .pio\build\teensy41\lib59c\Adafruit BMP3XX Library\bmp3.c.o
Compiling .pio\build\teensy41\libe7f\Adafruit GPS Library\Adafruit_GPS.cpp.o
Compiling .pio\build\teensy41\libe7f\Adafruit GPS Library\NMEA_build.cpp.o
Compiling .pio\build\teensy41\libe7f\Adafruit GPS Library\NMEA_data.cpp.o
Compiling .pio\build\teensy41\libe7f\Adafruit GPS Library\NMEA_parse.cpp.o
Archiving .pio\build\teensy41\lib144\libEWMA.a
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatDbg.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatFile.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatFilePrint.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatFileWrite.cpp.o
Archiving .pio\build\teensy41\lib631\libWire.a
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatFormatter.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatName.cpp.o
Archiving .pio\build\teensy41\libf97\libSPI.a
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatPartition.cpp.o
Archiving .pio\build\teensy41\libdd6\libAdafruit BusIO.a
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\ExFatVolume.cpp.o
Archiving .pio\build\teensy41\lib59c\libAdafruit BMP3XX Library.a
Compiling .pio\build\teensy41\lib088\SdFat\ExFatLib\upcase.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatDbg.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatFile.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatFileLFN.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatFilePrint.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatFileSFN.cpp.o
Archiving .pio\build\teensy41\libe7f\libAdafruit GPS Library.a
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatFormatter.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatName.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatPartition.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FatLib\FatVolume.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FreeStack.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FsLib\FsFile.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FsLib\FsNew.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\FsLib\FsVolume.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\MinimumSerial.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SdCard\SdCardInfo.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SdCard\SdSpiCard.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SdCard\SdioTeensy.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiArtemis.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiChipSelect.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiDue.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiESP.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiParticle.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiSTM32.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiSTM32Core.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\SpiDriver\SdSpiTeensy3.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FmtNumber.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FsCache.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FsDateTime.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FsGetPartitionInfo.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FsName.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FsStructs.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\FsUtf.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\PrintBasic.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\common\upcase.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\iostream\StdioStream.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\iostream\StreamBaseClass.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\iostream\istream.cpp.o
Compiling .pio\build\teensy41\lib088\SdFat\iostream\ostream.cpp.o
Compiling .pio\build\teensy41\lib413\SD\SD.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\AudioStream.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\CrashReport.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\DMAChannel.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\EventResponder.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial1.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial2.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial3.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial4.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial5.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial6.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial7.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial8.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\IPAddress.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\IntervalTimer.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Print.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Stream.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Time.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Tone.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\WMath.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\WString.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\analog.c.o
Compiling .pio\build\teensy41\FrameworkArduino\bootdata.c.o
Compiling .pio\build\teensy41\FrameworkArduino\clockspeed.c.o
Archiving .pio\build\teensy41\lib088\libSdFat.a
Compiling .pio\build\teensy41\FrameworkArduino\debugprintf.c.o
Compiling .pio\build\teensy41\FrameworkArduino\delay.c.o
Compiling .pio\build\teensy41\FrameworkArduino\digital.c.o
Compiling .pio\build\teensy41\FrameworkArduino\eeprom.c.o
Compiling .pio\build\teensy41\FrameworkArduino\extmem.c.o
Compiling .pio\build\teensy41\FrameworkArduino\fuse.c.o
Compiling .pio\build\teensy41\FrameworkArduino\interrupt.c.o
Compiling .pio\build\teensy41\FrameworkArduino\keylayouts.c.o
Compiling .pio\build\teensy41\FrameworkArduino\main.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\memcpy-armv7m.S.o
Archiving .pio\build\teensy41\lib413\libSD.a
Compiling .pio\build\teensy41\FrameworkArduino\memset.S.o
Compiling .pio\build\teensy41\FrameworkArduino\new.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\nonstd.c.o
Compiling .pio\build\teensy41\FrameworkArduino\pwm.c.o
Compiling .pio\build\teensy41\FrameworkArduino\rtc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent1.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent2.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent3.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent4.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent5.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent6.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent7.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent8.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEventUSB1.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEventUSB2.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_alloc_valid.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_calloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_free.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_hash.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_malloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_malloc_stats.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_pool.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_realloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_realloc_i.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_realloc_move.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_szalloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_util.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_zalloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\startup.c.o
Compiling .pio\build\teensy41\FrameworkArduino\tempmon.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_audio.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_desc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_flightsim.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_inst.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_joystick.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_keyboard.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_midi.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_mouse.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_mtp.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_rawhid.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_seremu.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_serial.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_serial2.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_serial3.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_touch.c.o
Compiling .pio\build\teensy41\FrameworkArduino\yield.cpp.o
Archiving .pio\build\teensy41\libFrameworkArduino.a
Linking .pio\build\teensy41\firmware.elf
Checking size .pio\build\teensy41\firmware.elf
Calculating size .pio\build\teensy41\firmware.elf
Building .pio\build\teensy41\firmware.hex
teensy_size: Memory Usage on Teensy 4.1:
teensy_size:   FLASH: code:94832, data:12428, headers:8448   free for files:8010756
teensy_size:    RAM1: variables:378048, code:91992, padding:6312   free for local variables:47936
teensy_size:    RAM2: variables:12384  free for malloc/new:511904
========================= [SUCCESS] Took 9.97 seconds =========================
```

## Issues

closes #109, closes #111, closes #106, closes #104, closes #120, closes #101, closes #107, closes #108, closes #103, closes #105, closes #99, closes #100

### CODE REVIEW

File: main/attitude.h [hunk 1]
```diff
@@ -64,7 +64,7 @@ Directional getAttitude(Directional gyro, bool hasLaunched){//only calibrated gy
   const int hz = 100; //number of readings per second
   static Directional oldChanges; //5-10 seconds ago
   static Directional lastChanges; //0-5 seconds ago
-  static int counter = 0;
+  static uint8_t counter = 0;
```
Comment (JonathanSafer) [inline main/attitude.h:67]: the maximum value of a `uint8_t` is 255, but we need this number to be able to go up to 500. Thus we should use `uint16_t` instead

File: main/main.ino [hunk 2]
```diff
@@ -39,17 +35,17 @@ void setup()
...
 
 void loop()
 {
-  static int tick = 0;
+  static uint32_t tick = 0;
   static flightPhase status = ONPAD;
-  unsigned int tickTime = getTickTime(status);
-  int x = 1;
+  uint32_t tickTime = getTickTime(status);
+  bool x = 1;
```
Comment (JonathanSafer) [inline main/main.ino:48]: if we're changing `x` to `bool`, it should be initialized with `true` instead of `1`

**Review by JonathanSafer — CHANGES_REQUESTED** [review]
Looks good, in the future you should try to address 1 issue per PR

File: main/attitude.h (hunk 1 above)
Comment (Sharkatron) [reply main/attitude.h]: should be fixed now. I noticed that `calibrateGyro` function had `uint8_t` for counter so I changed that to `uint16_t` as well.

Comment (JonathanSafer) [reply main/attitude.h]: awesome nice catch

**Review by JonathanSafer — APPROVED** [review]
Nice job!

**Pull request merged**